bamboo/unit_tests/test_unit_algo_ltfb.py

```diff
@@ -65,8 +65,7 @@ def setup_experiment(lbann):
     RPE = lbann.RandomPairwiseExchange
     SGD = lbann.BatchedIterativeOptimizer
     metalearning = RPE(
-        metric_name='random',
-        metric_strategy=RPE.MetricStrategy.HIGHER_IS_BETTER)
+        metric_strategies={'random': RPE.MetricStrategy.HIGHER_IS_BETTER})
```

Comment:
Can we test multiple metrics?

Reply:
We can. I'm working on some Catch tests. In the meantime, I will add a new Bamboo test for this feature. I don't want to modify this one, since this is supposed to mimic `test_unit_callback_ltfb.py`, at least until that callback has been removed.